jax/_src/numpy/lax_numpy.py

```diff
@@ -1037,6 +1037,10 @@ def histogramdd(sample, bins=10, range=None, weights=None, density=None):
   if weights is not None and weights.shape != (N,):
     raise ValueError("should have one weight for each sample.")
 
+  if range is not None and len(range) != D:
```

Comment:
In addition to checking that it's a sequence of length D, would it make sense to check that every element is itself a sequence of length 2 (of integers)? I wonder if `try: np.shape(range) == (N, D)` would work...

Reply:
Sure, I'll add that validation.